### ISSUE TITLE
Crash after password entry.  Some .tt files do not update, re-importing gpx is incomplete.

### ISSUE THREAD

**Issue by aleksejrs** [redfish64/TinyTravelTracker#28]
TTT 1.0.16.2 from F-droid crashes after I enter the password.  It collects data, but cannot show its UI.

AreaPanel.tt and TimeTree.tt have an older mtime (about when I noticed the problem) than the other files do.  If I delete these files, TTT works, but the range selector is showing only a very short rainbow.  I can then export a GPX file and it works for other programs.

If I delete everything and import the GPX file mentioned above, the resulting track ends early (IIRC, at a date a few days earlier than when the UI and .tt files broke).

A couple days before then, I changed GPS use frequency from about 10-25% to 100% and left it at that.

**Comment by aleksejrs** [comment]
TLDR: apparently everything I tried to solve the problem above actually worked, and there was nothing to report other than a UI confusion.

I replaced the files, and it started crashing again.
I renamed the .tt files, but it kept crashing.
I deleted everything but tile_cache and imported the GPX.  Now it works properly.

Actually, it works the same way it did after deleting the .tt files: “the range selector is showing only a very short rainbow” is because it shows a rainbow only for what is on the screen right now.  Maybe the track was ending “early” last time because I zoomed in to the wrong place.
